src/chimera/interfaces/telescope.py

```diff
@@ -417,6 +417,28 @@ def unparkComplete(self):
         successfully.
         """
 
+    def openCover(self):
```

Comment:
Ok. Done. I think that we should review the telescope interface (and probably the others too) to take in account the features like in the Camera Interface. In Telescope we would have to put the cover control as one of them.